data/net/minecraft/advancements/critereon/FilledBucketTrigger.mapping

```diff
@@ -3,13 +3,18 @@ CLASS net/minecraft/advancements/critereon/FilledBucketTrigger
 		ARG 1 json
 		ARG 2 entityPredicate
 		ARG 3 conditionsParser
+	METHOD lambda$trigger$0 (Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/advancements/critereon/FilledBucketTrigger$TriggerInstance;)Z
+		ARG 1 instance
 	METHOD trigger (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/item/ItemStack;)V
 		ARG 1 player
 		ARG 2 stack
 	CLASS TriggerInstance
+		METHOD <init> (Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/ItemPredicate;)V
+			ARG 1 player
+			ARG 2 item
 		METHOD filledBucket (Lnet/minecraft/advancements/critereon/ItemPredicate;)Lnet/minecraft/advancements/critereon/FilledBucketTrigger$TriggerInstance;
-			ARG 0 itemCondition
+			ARG 0 item
```

Comment:
`item` -> `itemPredicate`

Reply:
As above, these builders delegate to constructors, and their parameters match the field names there - `item`. Consistency with mojmap > naming directly from the class name.